diff --git a/added b/moved
similarity index 85%
rename from added
rename to moved
index a071991..4dbab21 100644
--- a/added
+++ b/moved
@@ -9,4 +9,4 @@ Some content
 Some content
 Some content
 Some content
-Some content
+Some modified content

diff --git a/oldfile b/newfile
similarity index 85%
rename from oldfile
rename to newfile
index a071991..4dbab21 100644
--- a/oldfile
+++ b/newfile
@@ -9,4 +9,4 @@ Some content
 Some content
 Some content
 Some content
-Some content
+Some modified content

diff --git a/sub/onefile b/sub/otherfile
similarity index 100%
rename from onefile
rename to otherfile
